doc/guide/packages.xml

```diff
@@ -253,4 +253,50 @@ mypackage/test.min.js.gz
 
   </section>
 
+  <section id="package-replace">
+    <title>Replacing an existing package</title>
+
+    <para>If the functionality in a package replaces one of the default packages
+      then it can replace that package by claiming the same <code>name</code> and a
+      higher <code>priority</code>.</para>
+
+    <para>For example a package in the <filename>/usr/share/cockpit/disks</filename>
+      directory should replace the <emphasis>storage</emphasis> package that comes
+      with Cockpit would have a <filename>manifest.json</filename> that looks like this:</para>
+
+<programlisting>
+{
+  "version": 0,
+  "name": "storage",
+  "priority": 10,
+  "menu": {
+     "index": {
+        "label": "Disk Storage",
+        "order": 15
+     }
+  }
+}
+</programlisting>
+
+    <para>It is also possible to hide or change labels on the menu items of an existing
+      package by including a <filename>override.json</filename> in that existing package's
+      directory.</para>
+
+    <para>For example an <filename>/usr/share/cockpit/systemd/override.json</filename> could
+      hide the <emphasis>Logs</emphasis> and move the <emphasis>Services</emphasis> menu item
```

Comment:
I suggest either `the <emphasis>Logs</emphasis> item` (or `entry`) or without `the`